format/midi/sysex.go

```diff
@@ -0,0 +1,157 @@
+package midi
+
+import (
+	"fmt"
+
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+var sysex = scalar.UintMapSymStr{
+	0x00f0: "Sysex Message",
+	0x80f7: "Sysex Continuation",
+	0x00f7: "Sysex Escape",
+}
+
+func decodeSysExEvent(d *decode.D, status uint8, ctx *context) {
+	ctx.running = 0x00
+
+	delta := func(d *decode.D) {
+		dt := d.FieldUintFn("delta", vlq)
+		d.FieldValueUint("tick", ctx.tick)
+
+		ctx.tick += dt
+	}
+
+	switch {
+	case status == 0xf0 && ctx.casio:
+		d.Errorf("SysExMessage F0 start byte without terminating F7")
+
+	case status == 0xf0:
+		d.FieldStruct("SysExMessage", func(d *decode.D) {
```

Comment:
Same as for `*MapSymStr` try to use jq-identifier if possible, ex `snake_case`. It's mostly to make queries use the jq code style

Reply:
(as above)